Assets/Scripts/SelectionSquare.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public class SelectionSquare : MonoBehaviour, IPointerClickHandler
+{
+    public GameObject my_placement_object;
+    public GameObject ui_handler;
+
+    public void Start()
+    {
+        ui_handler = GameObject.Find("UI_Manager");
```

Comment:
Game Object names can change, use a singleton MonoBehaviour if this is absolutely necessary and `FindObjectOfType<UIManager>()`

Reply:
fixed